docs/source/release/v6.12.0/Framework/Python/New_features/38144.rst

```diff
@@ -0,0 +1 @@
+- allows for declaring the many `PropertyWithValue` types as output properties from the python API.
```

Comment:
```suggestion
- allows for declaring the many :class:`PropertyWithValue <mantid.kernel.FloatPropertyWithValue>` types as output properties from the python API.
```
this will link to the wrong specific place, but a good enough reference